hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/upload/AbstractFileTransferHandler.java

```diff
@@ -299,7 +298,7 @@ private void streamToRepository() {
                 return Optional.ofNullable(artifactManagement.create(new ArtifactUpload(inputStream,
                         fileUploadId.getSoftwareModuleId(), filename, null, null, true, mimeType, -1)));
             } catch (final ArtifactUploadFailedException | InvalidSHA1HashException | InvalidMD5HashException e) {
-                LOG.error("Failed to transfer file to repository", e);
+                LOG.debug("Failed to transfer file to repository", e);
```

Comment:
Error cases should not logged on debug level but also not on error level. I suggest to log this error case on info level.